### PR TITLE
Update signal links to be consistent

### PR DESCRIPTION
**Pull request by svrnm**
via #2877 

> I get a section on "Reliability & Metrics" - cool. I know what metrics are, but I need to do a thing, and this isn't explaining that yet. Okay, I see another link in that section on "metrics", let's see where that goes.
That link goes to exactly where I am. Why is it a link? Well that's frustrating. Okay, now I'm on my own to work out my own adventure.

Our links for traces, metrics, logs are not consistent, they sometimes point to the observability primer sections (and from there they are even self-referential) and sometimes they point to the concept pages on traces, metrics, logs.

Personally I find the second more reasonable, that's why I replaced all the links to point to those individual concept pages.

### CODE REVIEW

**Review by cartermp — APPROVED** [review]

**Comment by cartermp** [comment]
Big love this one

File: content/en/docs/what-is-opentelemetry.md [hunk 1]
```diff
@@ -21,13 +21,13 @@ critical.
...
-then be sent to an Observability back-end. There are a number of Observability
-back-ends out there, ranging from self-hosted open source tools (e.g.
+must emit [traces](/docs/concepts/signals/traces/)),
+[metrics](/docs/concepts/signals/metrics/)), and
+[logs](/docs/concepts/signals/logs/)). The instrumented data must then be sent
+to an Observability back-end. There are a number of Observability back-ends out
+there, ranging from self-hosted open source tools (e.g.
 [Jaeger](https://www.jaegertracing.io/) and [Zipkin](https://zipkin.io/)), to
-commercial SaaS offerings.
+[commercial offerings](http://localhost:1313/ecosystem/vendors/).
```
Comment (chalin) [inline content/en/docs/what-is-opentelemetry.md:30]: You probably didn't been to include `http://localhost:1313`

**Review by chalin — COMMENTED** [review]
See inline comment.

File: content/en/docs/what-is-opentelemetry.md (hunk 1 above)
Comment (chalin) [reply content/en/docs/what-is-opentelemetry.md:30]: @svrnm ☝️

**Pull request merged**